data/factory.go

```diff
@@ -94,20 +94,20 @@ var TxFactory = [...]func() Transaction{
 }
 
 var ledgerEntryNames = [...]string{
-	ACCOUNT_ROOT:     "AccountRoot",
-	DIRECTORY:        "DirectoryNode",
-	AMENDMENTS:       "Amendments",
-	LEDGER_HASHES:    "LedgerHashes",
-	OFFER:            "Offer",
-	RIPPLE_STATE:     "RippleState",
-	FEE_SETTINGS:     "FeeSettings",
-	ESCROW:           "Escrow",
-	SIGNER_LIST:      "SignerList",
-	TICKET:           "Ticket",
-	PAY_CHANNEL:      "PayChannel",
-	CHECK:            "Check",
-	DEPOSIT_PRE_AUTH: "DepositPreAuth",
```

Comment:
Only really need to change the text string to make it work?

Reply:
Yes, when i try decode transaction data to struct i get error: Unknown LedgerEntryType: DepositPreauth

Reply:
I guess the point was -  you only really need to change the text string, with the horrible capitalisation, rather than all the more correct struct and const names.

Reply:
Okey, I'm fixed

Reply:
These should both have the same text string. After that it's all good :-)

https://github.com/rubblelabs/ripple/blob/master/data/factory.go#L109
https://github.com/rubblelabs/ripple/blob/master/data/factory.go#L126

Reply:
Oh, sorry, done)